game/configuration.cc

```diff
@@ -108,7 +108,11 @@ void ConfigItemXMLLoader::update(ConfigItem& item, xmlpp::Element& elem, unsigne
 			item.setLongDescription(getText(elem, "long"));
 		}
 		else {
-			const auto type = getAttribute(elem, "type");
+			std::string type{getAttribute(elem, "type")};
+			if (item.getType() == "uint" && type == "int") {
+				type = "uint"; // Convert old config values.
```

Comment:
Let's double check if this is the only type conversion or if we have some more.
I see we do define: `uint`, `int`,  and `float`

Reply:
Yes but I don't think I changed any other types. Checking the history of schema.xml should do the trick either way